languages/emacs-lisp/reference/progression.md

```diff
@@ -0,0 +1,11 @@
+# Emacs Lisp Concept Exercise Progression
+
+This is a working document to keep track of ideas and thoughts on how the progression through the Concept Exercises on the Emacs Lisp track could work.
+
+```mermaid
```

Comment:
is it normal that this is not rendered?

Reply:
It's rendered on the website, which can be [run locally](https://github.com/exercism/v3/blob/master/docs/maintainers/website.md).